src/include/pg_tde_defs.h

```diff
@@ -0,0 +1,16 @@
+/*-------------------------------------------------------------------------
+ *
+ * pg_tde_defs.h
+ * src/include/pg_tde_defs.h
+ *
+ *-------------------------------------------------------------------------
+ */
+#ifndef PG_TDE_DEFS_H
+#define PG_TDE_DEFS_H
+
+
+extern const char *pg_tde_package_string(void);
+extern const char * pg_tde_package_name(void);
```

Comment:
```suggestion
extern const char *pg_tde_package_name(void);
```

just for the formatting consistency